thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datalayer/bao/jdbc/MergedAnomalyResultManagerImpl.java

```diff
@@ -456,6 +456,24 @@ public void updateAnomalyFeedback(MergedAnomalyResultDTO entity) {
     return this.getAnomaliesForMetricBeanAndTimeRange(mbean, start, end);
   }
 
+  @Override
+  public MergedAnomalyResultDTO findParent(MergedAnomalyResultDTO entity) {
+    List<MergedAnomalyResultBean> candidates = genericPojoDao.get(Predicate.AND(
+        Predicate.EQ("detectionConfigId", entity.getDetectionConfigId()),
+        Predicate.LE("startTime", entity.getStartTime()),
```

Comment:
we may need to consider the potential merging gap between the parent and children anomaly.